src/utils/zcl_abapgit_convert.clas.abap

```diff
@@ -277,16 +263,8 @@ CLASS ZCL_ABAPGIT_CONVERT IMPLEMENTATION.
 
   METHOD xstring_to_int.
 
-    DATA: lv_xstring TYPE xstring,
-          lv_x       TYPE x.
-
-
-    lv_xstring = iv_xstring.
-    WHILE xstrlen( lv_xstring ) > 0.
-      lv_x = lv_xstring(1).
-      rv_i = rv_i * 256 + lv_x.
-      lv_xstring = lv_xstring+1.
-    ENDWHILE.
+* use the built-in type conversion
+    rv_i = iv_xstring.
```

Comment:
the old code didn't, but should it catch overflows?

Reply:
its only used for git protocol stuff, where the input is max 4 bytes hex, so it wont overflow

might end up sometime deleting this method altogether, moving it into the git package